ifrs17/Report/ParameterReportMutableScopeInteractive.ipynb

```diff
@@ -154,7 +154,7 @@
                 "\n    async Task<GridOptions> GetPartnerRatingReport()",
                 "\n    {",
                 "\n        await GetStorage().InitializeAsync((Year, Month), ReportingNode, Scenario, CurrencyType);",
-                "\n        var data = await workspace.GetPartnerRatingsReportParametersAsync(GetArgs());",
+                "\n        var data = await workspace.GetCurrentPartnerRatingsReportParametersAsync(GetArgs());",
```

Comment:
don't we have the equivalent for LockedIn?

Reply:
A separate PR will be created for the locked in interactive reports.